15/umbraco-cms/extending/packages/creating-a-package.md

```diff
@@ -148,60 +145,60 @@ The `Title`, `Description`, `PackageTags` came with the template and we added so
 | PackageProjectUrl        | https://umbraco.com | This URL will be shown as the package's URL when others install it. It will likely be a GitHub repository, or similar.                                                                                      |
 | PackageLicenseExpression | MIT                 | The license is set to MIT. Please consider how you want your package licensed. If in doubt when deciding an open-source license there are [good resources available](https://choosealicense.com/licenses/). |
 
-### Pack it
+### Pack the Package
 
-It is time to create the actual NuGet package (that is, a _.nupkg_ file). Executing the `dotnet pack` command in the package directory will take care of building the project and outputing the generated NuGet package in the `bin` folder (_the output on the CLI shows the full path to the `.nupkg` file_).
+To create the actual NuGet package, run the `dotnet pack` command in the package directory. The package will be output to the `bin` folder.
 
-{% hint style="info" %}
-If you want to specify the output location, just execute the following command instead:
+![Package output in `bin` folder](images/package-cli-command.png)
 
-```
+Alternatively, to specify the output location, use the following command:
+
+```cs
 dotnet pack --output MyNugetPackages
 ```
 
 It will pack the project in the current directory and place the resulting package into the `MyNugetPackages` folder.
```

Comment:
Does it make sense to mention or perhaps show what it should look like when it's done?
Like, did it create a file, and where? (if the specified location wasn't given).

Reply:
The cli image dispalys the file location. I haven't added a folder image cz it will get too confusing if the folders in the image are different from the user's folder.